### PR TITLE
Updated Versions

### PR DESCRIPTION
**Pull request by tuxfan**
- Added 7.1 release
- Added master git branch

### CODE REVIEW

File: var/spack/repos/builtin/packages/cgdb/package.py [hunk 1]
```diff
@@ -9,18 +9,30 @@
...
+    git      = 'https://github.com/cgdb/cgdb.git'
 
+    version('0.7.1', sha256='bb723be58ec68cb59a598b8e24a31d10ef31e0e9c277a4de07b2f457fe7de198')
     version('0.7.0', sha256='bf7a9264668db3f9342591b08b2cc3bbb08e235ba2372877b4650b70c6fb5423')
+    version('master', branch='master', submodule=False, preferred=False)
 
     # Required dependency
     depends_on('ncurses')
     depends_on('readline')
+    depends_on('autoconf', type='build')
```
Comment (junghans) [inline var/spack/repos/builtin/packages/cgdb/package.py]: Aren’t these just deps when building master.

File: var/spack/repos/builtin/packages/cgdb/package.py [hunk 2]
```diff
@@ -9,18 +9,30 @@
...
 
-    homepage = "https://cgdb.github.io"
-    url      = "https://cgdb.me/files/cgdb-0.7.0.tar.gz"
+    homepage = 'https://cgdb.github.io'
+    url      = 'https://cgdb.me/files/cgdb-0.7.1.tar.gz'
+    git      = 'https://github.com/cgdb/cgdb.git'
 
+    version('0.7.1', sha256='bb723be58ec68cb59a598b8e24a31d10ef31e0e9c277a4de07b2f457fe7de198')
     version('0.7.0', sha256='bf7a9264668db3f9342591b08b2cc3bbb08e235ba2372877b4650b70c6fb5423')
+    version('master', branch='master', submodule=False, preferred=False)
```
Comment (junghans) [inline var/spack/repos/builtin/packages/cgdb/package.py]: This should go on top.

**Review by junghans — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/cgdb/package.py (hunk 1 above)
Comment (tuxfan) [reply var/spack/repos/builtin/packages/cgdb/package.py]: Added.

File: var/spack/repos/builtin/packages/cgdb/package.py (hunk 2 above)
Comment (tuxfan) [reply var/spack/repos/builtin/packages/cgdb/package.py]: Done.

**Review by junghans — APPROVED** [review]

**Pull request merged**